grails-app/assets/javascripts/hubs.js

```diff
@@ -517,6 +517,7 @@ function ContentViewModel(config) {
     self.hideProjectGettingStartedButton = ko.observable(config.hideProjectGettingStartedButton || false);
     self.showIndigenousCulturalIPMetadata = ko.observable(config.showIndigenousCulturalIPMetadata || false);
     self.overriddenLabels = ko.observableArray();
+    self.showNespMetada = ko.observable(config.showNespMetada || false);
```

Comment:
can you refactor this and `showIndigenousCulturalIPMetadata` with `showCustomMetadata`? Thereby, all NESP metadata is controlled by one variable.

Reply:
what would be the label in the content page?
https://biocollect-staging.ala.org.au/nesp/admin/manageHubs

Reply:
show custom metadata such as category, raid, indigenous cultural etc.